js/script.js

```diff
@@ -1,57 +1,123 @@
+}
+
+function createAndAppend(title, author) {
+  // Create
+  const book = document.createElement('div');
+  const bookTitle = document.createElement('p');
+  const bookAuthor = document.createElement('p');
+  const removeBtn = document.createElement('button');
+  // const bookHr = document.createElement('hr');
+
+  // Classes
+  book.classList.add('book');
+  bookTitle.classList.add('book-title');
+  bookAuthor.classList.add('book-author');
+  removeBtn.classList.add('remove-btn');
+
+  // Add Content
+  bookTitle.textContent = `Title: ${title}`;
+  bookAuthor.textContent = `Author: ${author}`;
+  removeBtn.textContent = `Remove ${bookList.children.length + 1}`;
+
+  // Style
+  bookBackground(book, bookList.children.length % 2 !== 0);
+
+  // Append
+  book.append(bookTitle, bookAuthor, removeBtn);
+  bookList.append(book);
+
+  removeBtn.addEventListener('click', removeBook);
+}
+
+function loadContent() {
+  books = JSON.parse(localStorage.getItem('books'));
   books.forEach((book) => {
-    addBook(book.title, book.author);
+    createAndAppend(book.title, book.author);
   });
-};
+}
 
-if (localStorage.getItem('books')) {
-  books = JSON.parse(localStorage.getItem('books'));
+// To check if a book is repeated
+function checkTitleBook(book) {
+  return book.title === bookTitleInput.value;
 }
-const addBtn = document.getElementById('add-button');
-const addBtnHandler = (e) => {
-  e.preventDefault();
-  const bookTitleInput = document.getElementById('title');
-  const bookAuthorInput = document.getElementById('author');
-  const bookObject = {
-    title: bookTitleInput.value,
-    author: bookAuthorInput.value,
-  };
-  books.push(bookObject);
-  localStorage.setItem('books', JSON.stringify(books));
-  renderBooks(books);
-  bookTitleInput.value = '';
-  bookAuthorInput.value = '';
-};
-addBtn.addEventListener('click', addBtnHandler);
-renderBooks(books);
\ No newline at end of file
+
+class BookObject {
+  constructor(title, author) {
+    this.title = title;
+    this.author = author;
+  }
+
+  add() {
+    books.push(this);
+    localStorage.setItem('books', JSON.stringify(books));
+    createAndAppend(this.title, this.author);
+  }
+}
+
+function addBook() {
+  if (
+    bookTitleInput.value !== ''
+    && bookAuthorInput.value !== ''
+  ) {
+    if (books.some(checkTitleBook)) {
+      alertMessage.innerHTML = 'This book already exists';
+      alertMessage.style.display = 'block';
+      alertMessage.style.backgroundColor = 'rgb(165, 192, 200)';
+    } else {
+      const newBook = new BookObject(bookTitleInput.value, bookAuthorInput.value);
+      newBook.add();
+
+      bookTitleInput.value = bookTitleInput.defaultValue;
+      bookAuthorInput.value = bookAuthorInput.defaultValue;
+
+      alertMessage.style.display = 'none';
+    }
+  } else {
+    alertMessage.innerHTML = 'Add text to both fields';
+    alertMessage.style.display = 'block';
+    alertMessage.style.backgroundColor = 'rgb(185, 140, 228)';
+  }
```

Comment:
- [ ]  At the moment, this function for adding books is outside the class. Please kindly refactor code to have all functionality inside a class as defined in the project requirements.